### DISCUSSION TITLE
Switch游戏超分补帧

### DISCUSSION THREAD

**Discussion by Syaoran33** [hooke007/mpv_PlayKit#321] [Q&A 问答]
请问是否有对动漫游戏(或3D动漫游戏)较为适配的超分补帧配置文件

主要是通过采集卡将视频源输入到mpv,然后再进行超分补帧
输入的是1080p30fps，超分到2k60fps，显卡RTX3070移动端
ffmpeg/bin/ffmpeg.exe -list_devices true -f dshow -i dummy
start mpvnet/mpvnet.exe av://dshow:video="capturecardname" --profile=low-latency --untimed 
start ffmpeg/bin/ffplay -f dshow -i audio="audioname" -audio_buffer_size 4

游戏选择了塞尔达传说王国之泪和星之卡比探索发现
尝试了MPV_lazy预置的配置文件，发现效果不是很理想
超分效果没有动漫那么惊艳，画面像是锐化，超分效果不明显；
补帧倍帧延迟很大，显卡直接起飞了，尝试下来补帧文件就"MEMC_SVP_PRO.vpy"这个比较好用；
而且超分补帧几乎不能同时开启，画面输出延迟巨大

请问各位大佬有什么解决办法和建议吗，感谢感谢
对mpv开发方面了解的较少，可能提供的信息不够需要补充，请见谅

**Comment by hooke007** [comment]
我不觉得一个针对视频播放的方案对实时生成帧的游戏来说有任何参考价值。

视频补帧是通过已知的前后帧来插补中间帧，这完全不适合实时生成下一帧的游戏。

至于超分要想取得理想的表现那就更难了，现有的方案要么是针对真人视频，要么是针对2d动漫，3d游戏的场景几乎没有任何人涉及过。反倒是传统的算法(比如fsr)更有实用意义

**Reply by Syaoran33** [reply]
感谢大佬回复，想向你请教下
对于补帧我的理解可能有错误，假设游戏推流故意设定一段时间延后显示到播放器上，对延迟出来的一段视频流进行补中间帧，是否可以呢，这个延迟会需要很大吗

另外我看到一种运用在数字电视上的运动模糊补偿方法，通过AI尝试估计后续帧，现再有类似方案可以实现吗
memc技术：https://www.quora.com/What-is-the-MEMC-technology-in-television

**Reply by hooke007** [reply]
你现在做的补帧就是你说的东西，本质就是根据延迟的录像做后处理补帧，怎么处理延迟都是存在的，而且这些方案的设计目标是视频，这意味着延迟本身的宽容度很高。

至于那种预测未来型的memc我没见过。我用过的memc也只能基于现有帧补偿中间帧
估计后续帧的可靠依据是什么呢，只靠“图片”，它怎么知道你的下一步是向左还是向右呢？
对于游戏，我认为只有dlss3软硬结合深入游戏引擎的模式才是唯一解。

**Reply by Syaoran33** [reply]
明白了，感谢解答
